src/cript/api/vocabulary_categories.py

```diff
@@ -0,0 +1,29 @@
+# TODO consider getting these categories dynamically from the API
+all_controlled_vocab_categories = [
```

Comment:
I agree strongly with the TODO here.
We should not hard code these.
You the key list of the vocab instead.

Reply:
I currently do not have an API endpoint for this, but we can ask for one from @brili in the future and update the SDK with it

Reply:
If you query the vocab, you get a full dict, right?
So we can just use the keys of that dict, I thought.

Reply:
api = cript.API(host=None, token=None)

    # assertions
    assert api is not None
    assert isinstance(api, cript.API)
    api._fetch_vocab()

    assert sorted(cript.api.vocabulary_categories.all_controlled_vocab_categories) == sorted(api._vocabulary.keys())

Reply:
I am wrong. this is not fixable at the moment.